Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -235,6 +234,10 @@ def load_communities():
         if self.session.get_libtorrent():
             from Tribler.Core.Libtorrent.LibtorrentMgr import LibtorrentMgr
             self.ltmgr = LibtorrentMgr(self.session)
+            self.ltmgr.initialize()
+            # FIXME(lipu): upnp APIs are not exported in libtorrent python-binding.
```

Comment:
No, the funny thing is that libtorrent's documentation says to use this `add_port_mapping()` function. But it only exists in the C++ library, while the python-binding doesn't have it. If I use it with a libtorrent session, I will get a `something doesn't have this attribute...` error.

I guess perhaps they forgot to add it into the python-binding. I have already submitted an issue on libtorrent's bug tracker. Hope they get it fixed soon.